### PR TITLE
fix batch consuming issue

### PR DESCRIPTION
**Pull request by jialigit**
fix the batch checkpoint issue

# Description

fix batch consuming issue for checkpointing.

### CODE REVIEW

**Review by yiliuTo — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-spring-cloud-stream-binder-eventhubs/src/test/resources/application.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+spring:
```
[the remark this answers is not in the document]
Comment (jialigit) [reply sdk/spring/azure-spring-cloud-stream-binder-eventhubs/src/test/resources/application.yaml]: updated

**Review by yiliuTo — APPROVED** [review]
LGTM

**Pull request merged**